### ISSUE TITLE
Better handling of arrow keys

### ISSUE THREAD

**Issue by marco-silva0000** [oNaiPs/dmenu-mac#15]
I saw on the readme there was a todo for this, what exactly is needed?

While I was building it for my OS version, I made some changes, it basically wraps around the list wen going to the end

```
 if commandSelector == #selector(moveLeft(_:)) {
            if resultsText.selectedAppIndex > 0 {
                resultsText.selectedAppIndex -= 1
            }
            else {
                resultsText.selectedAppIndex = resultsText.list.count - 1
            }
            resultsText.updateWidth()
            return true
        } else if commandSelector == #selector(moveRight(_:)) {
            if resultsText.selectedAppIndex < resultsText.list.count - 1 {
                resultsText.selectedAppIndex += 1
            }
            else {
                resultsText.selectedAppIndex = 0
            }
            resultsText.updateWidth()
            return true
```

I've also changed the tab keypress to mimic the right key, but that's another issue...(I think tab and shift tab should mimic the arrow keys, key up should go to last, key down should go to first, and the "starts with" feature of the tab key should be replaced with the regex text, if the search string starts with ^)

**Comment by oNaiPs** [comment]
@marco-silva0000 thanks for helping! As long as what you're proposing more or less follows the original dmenu implementation, I can merge this if you create a PR with the changes. Thanks!

**Comment by oNaiPs** [comment]
This has been merged and released on 0.6.0. Closing
